gulp/isotest.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+"use strict";
+
+module.exports = (gulp, plugins, blueprint) => {
+    const rs = require("run-sequence").use(gulp);
+    const mocha = require("gulp-mocha");
+
+    blueprint.projectsWithBlock("isotest").forEach((project) => {
+        gulp.task(`isotest-${project.id}`, [`typescript-compile-${project.id}`], (done) => {
```

Comment:
@codebling if you use `blueprint.task()` you'll get the no-dependency behavior for free, though I wouldn't presume to make you figure out how that thing works. in short, it would generate `isotest-mocha-core` and `isotest-mocha-w-core`; the `-w-` task (short for "watch") runs without any dependencies.